app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
```

Comment:
The condition correctly checks if the command has three parts and starts with 'cp'. This is in line with the task description: 'Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.'